references/classification/utils.py

```diff
@@ -400,3 +401,57 @@ def reduce_across_processes(val):
     dist.barrier()
     dist.all_reduce(t)
     return t
+
+
+def set_weight_decay(
+    model: torch.nn.Module,
+    weight_decay: float,
+    norm_weight_decay: Optional[float] = None,
+    bias_weight_decay: Optional[float] = None,
+    custom_keys_weight_decay: Optional[Dict[str, float]] = None,
+):
```

Comment:
We should add documentation on this and provide examples for using it on `custom_keys_weight_decay`. From our offline chats I understand that this could be useful if you want to exclude arbitrary things (like positional embeddings) but we should clarify how/why here.